packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -150,6 +150,55 @@ class GestureRecognizerFactoryWithHandlers<T extends GestureRecognizer> extends
 /// ** See code in examples/api/lib/widgets/gesture_detector/gesture_detector.1.dart **
 /// {@end-tool}
 ///
+/// ## Troubleshooting
+///
+/// Why isn't my parent [GestureDetector] onTap method called?
+///
+/// Given a parent [GestureDetector] which defines an onTap callback and a child
+/// [GestureDetector] that also defines an onTap callback. When the inner
+/// [GestureDetector] is tapped, both [GestureDetector]s send a [GestureRecognizer]
+/// into the gesture arena because the pointer coordinates are in both
+/// [GestureDetector]s bounds. The child [GestureDetector] wins because it was
+/// the first detector to enter the arena and it got first dips because it is in front.
```

Comment:
s/dips/dibs/ but i agree with piinks that maybe we should be more explicit rather than using the concept of "dibs"